generator/whitelist.ts

```diff
@@ -322,6 +322,10 @@ const whitelist: WhitelistConfig[] = [
         basePath: 'vmwarevirtustream/resource-manager',
         namespace: 'Microsoft.VMwareVirtustream',
     },
+    {
+        basePath: 'visualstudio/resource-manager',
+        namespace: 'microsoft.visualstudio',
```

Comment:
Please could you update casing to "Microsoft.VisualStudio"?

Reply:
I have fixed it.